packages/did/src/DidDetails/LightDidDetails.spec.ts

```diff
@@ -400,6 +401,7 @@ describe('When creating an instance from a URI', () => {
       `did:kilt:light:00${validKiltAddress}:randomdetails`,
     ]
     incorrectURIs.forEach((uri) => {
+      // @ts-ignore
```

Comment:
This one is also warned.

Reply:
https://github.com/KILTprotocol/sdk-js/pull/505/commits/29da2de6f66b7e9f7251ca6af9f1d3a6fedc07b9